lua/class_definitions.lua

```diff
@@ -1518,46 +1518,13 @@ classes = {
     },
     mongroup = {
         attributes = {
-            posx = {
-                type = "int",
-                writable = true
-            },
-            posy = {
-                type = "int",
-                writable = true
-            },
-            posz = {
-                type = "int",
-                writable = true
-            },
-            tx = {
-                type = "int",
-                writable = false
-            },
-            ty = {
-                type = "int",
-                writable = false
-            },
-            dying = {
-                type = "bool",
-                writable = true
-            },
-            horde = {
-                type = "bool",
-                writable = true
-            },
-            diffuse = {
-                type = "bool",
-                writable = true
-            },
-            radius = {
-                type = "int",
-                writable = true
-            },
-            population = {
-                type = "int",
-                writable = true
-            }
+            pos = { type = "tripoint", writable = true },
+            target = { type = "tripoint", writable = false },
```

Comment:
Why is the target read-only, but the position is writable? Shouldn't it be the other way, the position is also used as key in the map of groups, so it should always match the value in the object.